client/packages/host/src/components/AppVersion.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC, CSSProperties } from 'react';
 import { Grid, Typography, useTranslation } from '@openmsupply-client/common';
 // Version is shared for client and server and is located in repo root package.json
-import packageJson from '../../package.json';
+import packageJson from '../../../../package.json'; // Get app version from "client" module
```

Comment:
Should be enough to update to `../../../../../package.json` - the package.json required is the one at the root. Though I do get a linting error when I try this! might need to look into that..?

Reply:
I also had linter error, it's outside of `rootDir` in typescript, I think would need to use `require('../../../../../package.json').version`

Reply:
It was lint error and a comment next to version property in package file made a bit confused about which package is the right one to update. Have now updated app version from right package file fd160ffdb